### ISSUE TITLE
Creating an account get a 10 liberapay coin free credit

### ISSUE THREAD

**Issue by MartinDelille** [liberapay/liberapay.com#1192]
Having a Liberapay specific coin when we get donations and give some back like explained in this [comment](https://github.com/liberapay/liberapay.com/issues/1171#issuecomment-410525102) by @ramack could help people using liberapay without having a specific bank reference.

This could be a good welcome invitation to credit every liberapay user of 10 coins. New user would also benefit this offer which could create an appeal.

**Comment by ramack** [comment]
The idea is nice, but I don't believe in it... Only exchanging "liberapay coins" is completely useless, so the coins would need a value like any currency and in this case the money would need to be there before you can  give new users free coins... And a it would become a game to create new liberapay accounts, just to get those free coins...

I fear it will not work, sorry.

**Comment by MartinDelille** [comment]
This money should not have euro equivalent would solve the problem you mentioned (Liberapay handles multiple currencies).

**Comment by ramack** [comment]
But then those coins would be just there to pass them around, without any benefit for anyone, aren't they?

**Comment by MartinDelille** [comment]
Yes like a step one in donation with a kind of virtual money. We could think of it as a game at the beginning.

We could also imagine we don't get only 10 free coins per account in a one shot but a monthly credit like a kind of basic income. (ping @Changaco).

**Comment by KindlyFire** [comment]
That would kind of be like the [Ğ1](https://duniter.org/en/introduction/) cryptocurrency... except it would be centralized and payments would be donations, if I get it right.

**Comment by MartinDelille** [comment]
It doesn't have to be crypto: just a simple token.

**Comment by MartinDelille** [comment]
@mathiasblc You down voted. Could you argue?

**Comment by KindlyFire** [comment]
> It doesn't have to be crypto: just a simple token.

It doesn't matter how you name it (at least to me). It's purpose would be to transfer value. 

Anyway, giving a valuable to all accounts on Liberapay on signup is doomed to fail. We'd have to implement some kind of ID checking, one person = one account, otherwise we are going to get tons of signups from the same people, just inflating their wallet, to later cash out. This is hard to do, more so because we are a non-profit (limited funds, or more like no funds right now).

**Comment by MartinDelille** [comment]
There is many way to check a real user identity:

- A blockchained based identification protocol
- Github connect with a public contribution historic
- Facebook connect to an account with a minimum number of friends
- Twitter connect with a minimum amount of followers
- ...

**Comment by kindlyfire** [comment]
All can be faked without too much trouble.

**Comment by MartinDelille** [comment]
Facebook requires to upload a copy of the user password for certification issue. I guess this is not so easy to fake.

**Comment by MartinDelille** [comment]
And a github user contributing to opensource project is a quite reasonnable way to ensure that he is a real person (proof of work).

**Comment by kindlyfire** [comment]
> Facebook requires to upload a copy of the user password for certification issue. I guess this is not so easy to fake.

Can you explain ?

> And a github user contributing to opensource project is a quite reasonnable way to ensure that he is a real person (proof of work).

If the process of verifying is automated, it is easy to circumvent. We'd spend a lot of man-hours verifying people's identity if we did it manually.

**Comment by MartinDelille** [comment]
I didn't wanted to write *certification* but *authentification*. Sometimes, facebook blocks user and ask for a scan of their passport.

**Comment by kindlyfire** [comment]
That could be used if it is reliable, but that still begs the question: why would a libre platform rely on an internet giant for some of its functionality, and the fact that some users will be excluded by the mere fact that they do not have and do not want a Facebook account (why would Liberapay force me to create one ?)

**Comment by MartinDelille** [comment]
Of course we should not rely on internet giant. Facebook was just an example that it is possible to verify identity on the internet (at least at a basic level that would discourage a bad intention guy). 

After that what really happen if someone create multiple accounts ? Liberapay isn’t an exchange plateform so the token wouldn’t have a real value.

**Comment by mathiasblc** [comment]
@MartinDelille maybe I’m a bit late to answer you but, anyway... I feel really sick everytime some techbro throw bad crypto idea in the basket. We don’t need this shit, we don’t want it. As the thread goes on I feel even more bad, reading this suggestions about GitHub or Facebook third-party auth.

Please don’t mention me here again.

**Comment by MartinDelille** [comment]
I feel kind of hurt to be downvoted and poorly argumented by people who doesn't look so active on the Liberapay's project. I would prefer to have an open discussion here:

- if you are not interested, just go away.
- if you have a suggestion or a better idea, feel free to comment so we can evolve to something better.

If you're not interested in a part of the project, you can just not use this part and everyone will be happy! 😄 #bisounourspointreach

**Comment by MartinDelille** [comment]
For your information, I'm more or less involved in the project since the begining and opened the discussion about blockchain just a few weeks after it was alive: https://github.com/liberapay/salon/issues/14

You may not like it, I do.

**Comment by mattbk** [comment]
> if you are not interested, just go away.
> if you have a suggestion or a better idea, feel free to comment so we can evolve to something better.

What if the better idea is to work on getting basic functionality back before throwing imaginary money into the mix?

If the point of Liberapay is to make it easy for people to pay real money to other people/groups so they can be sustainably funded, how is adding imaginary money to the platform helping?

What will make the imaginary money have value? Liberapay provides a money transfer service--will the imaginary money be able to be traded to Liberapay for enhanced service of some sort?

As for preventing people from gaming the system--the concerns others have listed are valid, and furthermore encroach on a user's right to anonymity.

**Comment by MartinDelille** [comment]
> What if the better idea is to work on getting basic functionality back before throwing imaginary money into the mix?

You're right. There is no urgency to implement this kind of idea. 

But by my experience, it is easier to use a plateforme that doesn't deal with real money.

This imaginary money (let's call it libcoin) would be like a game: giving it to project that you support would be like liking a post on facebook or claping a medium post: It doesn't help the creator to buy beer but it's always good to feel that we are supported!

**Comment by slinkeepy** [comment]
Life is complicated enough...
Also anything with even a _passing resemblance_ to crypto should be a hard "no".

p.s. I can see why there are 500 open issues here -- nothing gets closed! :X
